packages/fxa-settings/src/pages/Signin/SigninRecoveryMethod/Phone/ConfirmRecoveryCode/container.tsx

```diff
@@ -0,0 +1,45 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useState } from 'react';
+import { RouteComponentProps } from '@reach/router';
+import ConfirmRecoveryCode from '.';
+import { ResendStatus } from '../../../../../lib/types';
+
+const ConfirmRecoveryCodeContainer = (_: RouteComponentProps) => {
+  const [errorMessage, setErrorMessage] = useState('');
+  const [resendErrorMessage, setResendErrorMessage] = useState('');
+  const [resendStatus, setResendStatus] = useState<ResendStatus>(
+    ResendStatus.none
+  );
+
+  const verifyCode = async (otpCode: string) => {};
+  const resendCode = async () => {};
+
+  const clearBanners = () => {
+    setErrorMessage('');
+    setResendErrorMessage('');
+    setResendStatus(ResendStatus.none);
+  };
+
+  // TODO: get from api
+  // const account = useAccount();
+  const maskedPhoneNumber = '***-***-1234';
```

Comment:
Tiny nit, you could go ahead and set this to the bullet points similar to: https://github.com/mozilla/fxa/pull/18180/files#diff-dfe1f11a80c8747bf0d5789fabe7a440e4296ccb5c8da58d2dfd962fe4bc62a3R42